### PR TITLE
Changelog for v0.18.7

### PR DESCRIPTION
**Pull request by taneyland**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/whatsnew/changelog.md [hunk 1]
```diff
@@ -21,6 +21,22 @@ description: >
...
+### Tool Upgrade
+- EKS Anywhere v0.18.7 Admin AMI with CVE fixes for Amazon Linux 2
+
+### Supported Operating Systems
+
+|                     | vSphere | Bare Metal | Nutanix | CloudStack | Snow  |
+| :----------:        | :-----: | :--------: | :-----: | :--------: | :---: |
+| Ubuntu 20.04        | ✔       | ✔          | ✔       | —          | ✔     |
+| Ubuntu 22.04        | ✔       | ✔          | ✔       | —          | —     |
+| Bottlerocket 1.15.1 | ✔       | ✔          | —       | —          | —     |
```
Comment (abhay-krishna) [inline docs/content/en/docs/whatsnew/changelog.md:36]: This should be 1.19.0. We bumped it in EKS-A 0.18.6.

Comment (taneyland) [reply docs/content/en/docs/whatsnew/changelog.md]: Thanks for catching that. Updated

**Pull request merged**